.pre-commit-config.yaml

```diff
@@ -10,3 +10,22 @@ repos:
       - id: check-byte-order-marker
       - id: check-executables-have-shebangs
       - id: check-merge-conflict
+  - repo: https://github.com/openstack-dev/bashate.git
+    rev: "0.6.0"
+    hooks:
+      - id: bashate
+        entry: bashate --error . --ignore=E003,E002,E006,E011,E020,E041,E042
```

Comment:
Please make this part of the Makefile (i.e., `make validate`). I don't want to play ping-pong with the CI but really want a local make target to run tests.

Reply:
By having it in `make validate`, running the linter will then automatically be part of the gating task in Cirrus.

Reply:
@edsantiago In regards to `.bash` extension I think you will really struggle to defend the case that this is an extension owned by bats tools. Yes, is a bug, but not in bashate, is a bug in bats that makes this owneship assumpts. They could as well use `.bats` instead which does not overlap with others.

Sure I can replace the sheband with another variant, but I am kinda become tired of nit-picking. The review allows edits from others and they are more than welcome. As far we finish it faster the better, already spend more time on it than expected.

Now I do have some questions regarding the complex nature of the makefile and how is used on CI. Maybe someone can build a mermaid diagram with the dependencies?

My expectation was that CI would run `make lint` somewhere, preferably as soon as possible. Most projects have lint job but should I guess that libpod one is called validate, right?